### ISSUE TITLE
Captive portal not accessible sometimes

### ISSUE THREAD

**Issue by gjt211** [alanswx/ESPAsyncWiFiManager#6]
I am using the following;
1. ESPAsyncTCP
2. ESPAsyncWebServer
3. ESPAsyncWifiManager
4. 2.4.0-rc1 ESP8266 core
5. NodeMCU board 12E variant
6. Arduino IDE 1.8.3 on Mac OS
All libraries are as of today the 27th August 2017.

On a newly programmed ESP8266, or after I do a `WiFi.disconnect();` or `wifiManager.resetSettings();` the serial terminal shows me that I am running in access point mode...
```
*WM: HTTP server started
```
Then after about 2 seconds I get this
```
*WM: About to scan()
*WM: About to scan()
``` 
then at about 5 seconds it prints
```
*WM: Scan done
```
And then after about 12 seconds it repeats the above until my configPortalTimeout, or until I try to connect to it.
```
*WM: HTTP server started
*WM: About to scan()
*WM: About to scan()
*WM: Scan done
*WM: About to scan()
*WM: About to scan()
*WM: Scan done
*WM: About to scan()
*WM: About to scan()
*WM: Scan done
```

I then have random success connecting to it as an AP. Sometimes it connects and gives me a blank captive portal page. Sometimes it won't connect. Sometimes it will connect but won't let me save.

At the start of my code I have
```
AsyncWebServer server(80);
DNSServer dns;
```
I am using a timeout as follows;
```
wifiManager.setConfigPortalTimeout(180);
```

I am using the following so I can start and stop EspAsyncMQTT
```
WiFiEventHandler wifiConnectHandler;
WiFiEventHandler wifiDisconnectHandler;

void onWifiDisconnect(const WiFiEventStationModeDisconnected& event) {
  Serial.println(F("[WIFI] {DIS} Disconnected from Wi-Fi."));
  mqttReconnectTimer.detach(); // ensure we don't reconnect to MQTT while reconnecting to Wi-Fi
}

void onWifiConnect(const WiFiEventStationModeGotIP& event) {
  Serial.println(F("[WIFI] {CON} Connected to Wi-Fi."));
  if (WiFi.isConnected()) {
    mqttReconnectTimer.once(2, connectToMqtt);
    Serial.println(F("[MQTT] {CON} Re-connecting"));
  }
}
```

I have been using the non Async version for a long time successfully on many projects so I don't really know what's going on here.

**Comment by alanswx** [comment]
Are you running out of memory? I haven't touched this in a long time, I think one of the other libraries might have changed.

**Comment by gjt211** [comment]
Hi @alanswx, thanks for your response. I will do some memory/heap tests later today and report back. I am just a little tied up as WHM/cPanel daily update has caused me some headaches today with MySQL and JSON.

**Comment by gjt211** [comment]
Hi @alanswx, sorry for taking so long to get back to this.
Free heap never goes below about 22000 throughout all my code that's with various sensors, asyncMqtt and other stuff in there. It is normally over 30000 before I start the SSL MQTT stuff. So I don't think it is a memory issue.
I have just forked your repo and added a couple of small changes that I always seem to make (change the `<h1></h1>` to a `<h2></h2>` for the AP name as it fits better on a mobile screen) along with some simple text changes.
I noticed a recent merge #43 and will give that a go this week to see how I go, and then reply with my findings.
